REST/RESTListener+Replicate.cc

```diff
@@ -289,17 +311,19 @@ namespace litecore { namespace REST {
                          canceled ? "Stopped" : "No matching task");
             return;
         }
-
+        // Auth:
+        slice authType = params["authtype"].asString();
+        if (authType) {
```

Comment:
If I remember, only the basic auth type uses user and password.  If this is the only supported auth method in here then it should probably be compared accordingly.

Reply:
yes, replied earlier. Asked advice of dealing with the else part.

Reply:
If `authType` is anything other than `"Basic"`, return a 401 status.